nexus/src/app/instance.rs

```diff
@@ -56,6 +56,16 @@ impl super::Nexus {
             )));
         }
 
+        // Reject instances where the memory is not at least one gibibyte
+        if params.memory.to_whole_gibibytes() < 1 {
```

Comment:
To close out https://github.com/oxidecomputer/omicron/issues/1030 , could we do a granularity check too?